src/content/app/tools/blast/views/blast-submissions/BlastSubmissions.scss

```diff
@@ -1,7 +1,16 @@
 @import 'src/styles/common';
 
+.submissionsContainer {
+  display: grid;
+  grid-template-rows: auto auto minmax(0, 1fr);
+  height: 100%;
+}
+
 .container {
   padding: 0 30px $global-padding-bottom 60px; // TODO: these 30px, 60px and 90px should either become a reusable constant or be included in some sort of BlastMainContainer component
+  overflow: auto;
+  scroll-behavior: smooth;
```

Comment:
Why do you need this?

Reply:
copied from blastForm.scss :) i thought it makes a difference with the scrolling, not sure if its needed

Reply:
We have javascript code that scrolls to the bottom input box there.